caktus_primer/static/stylus/colors.styl

```diff
@@ -162,11 +167,23 @@ applyColorSet(pname, mode="content")
 	if not check-aa-contrast(getColorSetColor(pname, "bg"), getColorSetColor(pname, "fg"))
 		$skipAA = bool($csSkipAA[_getColorSetName(pname)])
 		if not $skipAA
-			error("Palette "+pname+" fails AA constrast requirements (4.5). Contrast was " + contrast(getColorSetColor(pname, "bg"), getColorSetColor(pname, "fg")).ratio)
+			error("Palette "+pname+" (fg) fails AA constrast requirements (4.5). Contrast was " + contrast(getColorSetColor(pname, "bg"), getColorSetColor(pname, "fg")).ratio)
 
 	if mode == "interactive" or 'interactive' in pname or $colorsets[pname].interactive
 		&:focus, &:hover
 			background-color: getColorSetColor(pname, "hover")
 		&:active
 			background-color: getColorSetColor(pname, "active")
-			transition: all 0s
\ No newline at end of file
+			transition: all 0s
+	else
+		a
+			text-decoration: none
+			border-style: dotted
+			border-width: 1px
+			border-color: transparent
+			border-bottom: 1px dotted
+			box-sizing: border-box
+		a, a:active, a:focus, a:hover
+			color: getColorSetColor(pname, "link")
```

Comment:
Correct me if I'm wrong here, but looking at the definition of `getColorSetColor`, the second parameter is supposed to be a `cname` that could potentially match `"hover"`, `"active"`, and so on.

What you're doing here makes sense for the styles for `a` … not so much for `a:active`, etc, which I'm guessing would want to use `getColorSetColor(pname, "active")` and so on—yes? Or do you specifically want to avoid doing that for links?

Reply:
Those other colors were made for buttons and other elements. To avoid difficult-to-meet contrast guidelines we opted for non-color designations on links.